desktop/src/main/java/pmstation/integration/Mqtt.java

```diff
@@ -57,15 +53,36 @@ public void messageArrived(String topic, MqttMessage message) {
                 @Override
                 public void deliveryComplete(IMqttDeliveryToken token) {}
             });
+        } catch (MqttException e) {
+            logger.error("Unable to create a Mqtt Client", e);
+        }
+    }
+
+    private void connect() {
+        String username = Config.instance().to().getString(Config.Entry.MQTT_USERNAME.key());
+        String password = Config.instance().to().getString(Config.Entry.MQTT_PASSWORD.key());
+
+        MqttConnectOptions options = new MqttConnectOptions();
+        options.setCleanSession(true);
+
+        if (!StringUtils.isEmpty(username) && !StringUtils.isEmpty(password)) {
+            options.setUserName(username);
+            options.setPassword(password.toCharArray());
+        }
+
+        try {
             client.connect(options);
             client.subscribe(topic);
         } catch (MqttException e) {
-            logger.error("Unable to create a Mqtt Client", e);
+            logger.error("Unable to connect/subscribe with the Mqtt Server", e);
```

Comment:
1.  with the -> to
2.  under server name (and port) to that error log
3. Mqtt -> MQTT